Software/Velvet_ESP32/Readme.md

```diff
@@ -0,0 +1,45 @@
+### Velvet_ESP32_STM_UART
+   Проект ESP32_STM_UART предназначен для реализации протокола общения между микроконтроллерами ESP32 и STM32
```

Comment:
Fix project name everywhere
```suggestion
### Velvet_ESP32
   Проект ESP32 предназначен для реализации протокола общения между микроконтроллерами ESP32 и STM32
```

Reply:
Done. But I don't wanna to rename the library